ntoskrnl/cc/copy.c

```diff
@@ -381,6 +381,16 @@ CcCanIWrite (
         return FALSE;
     }
 
+    /* Otherwise, if there are no deferred writes yet, start the lazy writer */
+    if (IsListEmpty(&CcDeferredWrites))
+    {
+        KIRQL OldIrql;
```

Comment:
Not sure you need this local variable, since there is already one available in the function, and the previous acquire-release spinlock has been already done (see lines 331-354).